pcsx2/Frontend/InputManager.cpp

```diff
@@ -933,19 +936,36 @@ bool InputManager::PreprocessEvent(InputBindingKey key, float value, GenericInpu
 
 void InputManager::GenerateRelativeMouseEvents()
 {
+	static float pos[2] = {0.0f, 0.0f};
```

Comment:
Probably should put this at the top of the file (similar to the other variables), just for consistency.

Reply:
ok, done that :)